tutor/src/screens/teacher-gradebook/index.js

```diff
@@ -23,6 +23,7 @@ const titleControls = (ux) => {
 
 const StyledTourRegion = styled(TourRegion)`
   display: flex;
+  flex-flow: column;
 `;
```

Comment:
Usually I see flex-flow with 2 arguments, like `flex-flow: column wrap`, since you're not specifying the 2nd argument for wrap behavior, this is equivalent to `flex-direction: column`